### PR TITLE
Update SQL doc to reflect changes in the APIs

### PR DESCRIPTION
**Pull request by akedin**
BeamSql.querySimple() and queryMulti() were combined into query().
BeamRecord was renamed to Row. Factory methods and builders were added to it.

### CODE REVIEW

**Comment by akedin** [comment]
R: @kennknowles @apilloud

File: src/documentation/dsls/sql.md [hunk 1]
```diff
@@ -14,97 +14,141 @@ This page describes the implementation of Beam SQL, and how to simplify a Beam p
 
 ## 1. Overview {#overview}
 
-SQL is a well-adopted standard to process data with concise syntax. With DSL APIs (currently available only in Java), now `PCollection`s can be queried with standard SQL statements, like a regular table. The DSL APIs leverage [Apache Calcite](http://calcite.apache.org/) to parse and optimize SQL queries, then translate into a composite Beam `PTransform`. In this way, both SQL and normal Beam `PTransform`s can be mixed in the same pipeline.
+SQL is a well-adopted standard to process data with concise syntax. With DSL APIs (currently available only in Java), now `PCollections` can be queried with standard SQL statements, like a regular table. The DSL APIs leverage [Apache Calcite](http://calcite.apache.org/) to parse and optimize SQL queries, then translate into a composite Beam `PTransform`. In this way, both SQL and normal Beam `PTransforms` can be mixed in the same pipeline.
```
Comment (melap) [inline src/documentation/dsls/sql.md]: Drive by comment -- if you end up making another commit for this PR, could we axe the * TOC {:toc} lines at the top of the page? We didn't have the right nav at the time this was added. Due to the bolding of a couple of the deeply nested sections, it looks a bit strange also.

Comment (akedin) [reply src/documentation/dsls/sql.md]: Removed

**Comment by kennknowles** [comment]
retest this please

**Comment by kennknowles** [comment]
One next step would be to use snippets capabilities to make sure these keep compiling.

**Comment by melap** [comment]
May need to be merged manually, as mergebot is having troubles right now.

**Comment by melap** [comment]
@asfgit merge

**Comment by asfgit** [comment]
Error: Custom preparation command failed. Check logs for more information.

**Comment by melap** [comment]
@asfgit merge

**Pull request closed without merging**